docs/manage/server-admin-gui.md

```diff
@@ -2,42 +2,62 @@
 
 # PMM Settings Page
 
-The *PMM Settings* page lets you configure a number of PMM options. It can be accessed through the main menu:
+The *PMM Settings* page lets you configure a number of PMM options. It can be accessed from the main menu:
 
 ![image](../_images/pmm-add-instance.png)
 
-## Settings
+!!! note
+
+    Press *Apply changes* to store any changes made in PMM Settings.
 
-The *Settings* section allows you to change [metrics resolution](../faq.md#metrics-resolution), [data retention](../faq.md#data-retention), as well as configure telemetry and automatic checking for updates:
+## Metrics resolution
 
-![image](../_images/pmm.settings_settings.png)
+Metrics are collected at three intervals representing low, medium and high resolutions. Short time intervals are regarded as high resolution metrics, while those at longer time intervals are low resolution.
 
-Press *Apply changes* to store any changes.
+![image](../_images/PMM_Settings_Metrics_Resolution.png)
 
-## Metrics resolution
+The *Metrics Resolution* radio button lets you select one of four presets.
+
+- *Rare*, *Standard* and *Frequent* are fixed presets.
+- *Custom* is an editable preset.
+
+Each preset is a group of Low, Medium and High metrics resolution values. Low resolution intervals *increases* the time between collection, resulting in low-resolution metrics and lower disk usage. High resolution intervals *decreases* the time between collection, resulting in high-resolution metrics and higher disk usage.
+
+The default values for the fixed presets are:
 
-Metrics are collected at three intervals representing low, medium and high resolutions.
-Short time intervals are regarded as high resolution metrics, while those at longer time intervals are low resolution.
+**Rare**
 
-The default values are:
+- Low: 300 seconds
+- Medium: 180 seconds
+- High: 60 seconds
 
-* Low: 60 seconds
-* Medium: 10 seconds
-* High: 5 seconds
+**Standard**
 
-The *Metrics Resolution* slider lets you choose from three preset combinations of intervals corresponding to high, medium, and low resolution (short, medium, and long collection periods).
+- Low: 60 seconds
+- Medium: 10 seconds
+- High: 5 seconds
 
-The slider tool-tip shows the collection time corresponding to each resolution setting.
+**Frequent**
 
-* Setting the slider to *Low* increases the time between collection, resulting in low-resolution metrics (and lower disk usage).
+- Low: 30 seconds
+- Medium: 5 seconds
+- High: 1 second
 
-* Setting the slider to *High* decreases the time between collection, resulting in high-resolution metrics (and higher disk usage).
+Values for the *Custom* preset can be entered as values, or increased or decreased using the up/down arrows.
```

Comment:
Is the the double "or" ok? Or is it better like "...values, or increased/decreased using the..."?

Reply:
Yes, a bit clumsy. I'll change it.